code/game/MapData/shuttles/nanotrasen_powerrangers.dm

```diff
@@ -0,0 +1,373 @@
+	name = "Loss Prevention Engineering Hardsuit Helmet"
+	desc = "The helmet that comes attached to the LP Team Engineering Hardsuit."
+	icon_state = "hardsuit0-ert_engineer"
+	item_state = "hardsuit0-ert_engineer"
+	hardsuit_type = "ert_engineer"
+
+/obj/item/clothing/suit/space/hardsuit/ert/lp/med
+	armor = list("melee" = 30, "bullet" = 15, "laser" = 15, "energy" = 20, "bomb" = 10, "bio" = 100, "rad" = 60, "fire" = 60, "acid" = 75)
+	helmettype = /obj/item/clothing/head/helmet/space/hardsuit/ert/lp/med
+	name = "Loss Prevention Medical Hardsuit"
+	desc = "The best of the best medical staff get assigned to the ERT. Second best are given this Hardsuit as a part of the LP Team."
+	icon_state = "ert_medical"
+	item_state = "ert_medical"
+
+/obj/item/clothing/head/helmet/space/hardsuit/ert/lp/med
+	armor = list("melee" = 30, "bullet" = 15, "laser" = 15, "energy" = 20, "bomb" = 10, "bio" = 100, "rad" = 60, "fire" = 60, "acid" = 75)
+	name = "Loss Prevention Medical Hardsuit Helmet"
+	desc = "The helmet that comes attached to the LP Team Medical Hardsuit."
+	icon_state = "hardsuit0-ert_medical"
+	item_state = "hardsuit0-ert_medical"
+	hardsuit_type = "ert_medical"
+
+/obj/item/clothing/head/helmet/space/hardsuit/ert/lp/clown
+	name = "LP Funny Hardsuit Helmet"
+	desc = "Every dog has its day, and every Funny Hardsuit has a Funny Helmet attached to it."
+	armor = list("melee" = 10, "bullet" = 5, "laser" = 10, "energy" = 10, "bomb" = 75, "bio" = 100, "rad" = 15, "fire" = 75, "acid" = 50)
+	icon_state = "hardsuit0-ert_clown"
+	item_state = "hardsuit0-ert_clown"
+	hardsuit_type = "ert_clown"
+
+/obj/item/clothing/suit/space/hardsuit/ert/lp/clown
+	name = "LP Funny Hardsuit"
+	desc = "Only the most faithful of those in service to the Honkmother make it anywhere aside from the mass driver. Let alone the LP."
+	armor = list("melee" = 10, "bullet" = 5, "laser" = 10, "energy" = 10, "bomb" = 75, "bio" = 100, "rad" = 15, "fire" = 75, "acid" = 50)
+	icon_state = "ert_clown"
+	item_state = "ert_clown"
+	helmettype = /obj/item/clothing/head/helmet/space/hardsuit/ert/lp/clown
+
+/obj/item/clothing/head/helmet/space/hardsuit/lp
+	name = "RIG heatsuit helmet"
+	desc = "The helmet to the RIG heat suit. It's packed with heat diverting materials, coolant pipes, and a two inch thick face screen."
+	armor = list("melee" = 1, "bullet" = 1, "laser" = 1, "energy" = 1, "bomb" = 50, "bio" = 100, "rad" = 100, "fire" = 100, "acid" = 75)
+	heat_protection = HEAD
+	max_heat_protection_temperature = FIRE_IMMUNITY_MAX_TEMP_PROTECT
+	resistance_flags = ACID_PROOF | FIRE_PROOF
+
+/obj/item/clothing/suit/space/hardsuit/ancient/lp
+	name = "RIG heat suit"
+	desc = "A fully heat resistance suit based on an early RIG hardsuit prototype. It sacrifices armor of any kind for intricate heatsinks. It remains rather bulky as a result."
+	armor = list("melee" = 1, "bullet" = 1, "laser" = 1, "energy" = 1, "bomb" = 50, "bio" = 100, "rad" = 100, "fire" = 100, "acid" = 75)
```

Comment:
why 1

Reply:
Because its a maxtemp immune suit, it serves no other purlose than to withstand that. It has almost no defense in exchange for that property.

Reply:
Then do 0